test/integration/targets/mysql_replication/tasks/mysql_replication_initial.yml

```diff
@@ -44,12 +46,13 @@
     master_password: "{{ replication_pass }}"
     master_log_file: "{{ master_status.File }}"
     master_log_pos: "{{ master_status.Position }}"
+    master_ssl_ca: _
```

Comment:
This still seems too much like a secret code rather than a friendly interface. It would be better to allow this field to accept standard values to indicate the field should be set to an empty string, such as `''`, `none`, or `~`. This also will look odd when someone comes across this task in the future to edit the playbook. It's not obvious what this does without reading the module documentation.

Reply:
I agree, but the problem is that we need to be able to pass empty values to any parameters. If you look at the code there is a mix of if x is not None / if x / etc. i’ll try again but wouldn’t like to make a mistake for one of them and better to avoid the situation when users get something unexpected after release:)

Reply:
@samdoran , PTAL, it will work, i think, nothing failed during the tests